_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,89 @@
+---
+name: Image accessible name is filename
+description: |
+  This rule checks that image elements that use their source filename as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to HTML `input type="image"` or `area` elements, or any HTML element with the semantic role of `img`, where the element is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an [accessible name](#accessible-name) that matches the filename specified in the `src` attribute.
+
+**Note**: URL query strings, difference in letter casing, and forward and trailing whitespace should be ignored. For example `src="/foo/bar.jpg?baz "` has a filename `bar.jpg`.
+
+### Expectation
+
+Each test target has an accessible name that can be used in place of the [non-text content](https://www.w3.org/TR/WCAG21/#dfn-non-text-content) of the element without loss of information to the user.
```

Comment:
I'm not sure about "loss of information" but don't have a better suggestion. The SC says "serves equivalent purpose".